README.md

```diff
@@ -37,6 +40,21 @@ Remember, however, take note of the major changes in releases greater than 0.12
 
 Climb aboard and explore the [Getting Started Guide][Getting Started] - you'll soon be sailing!
 
+## _NOTE: Draft is experimental and does not have a stable release yet._
+If you're having issues after upgrading draft, try deleting the draft home directory where all the draft configuration files live and then re-run the `draft init` command.
+Example:
+```console
+$ draft home
+/Users/admin/.draft
+$ rm -rf $(draft home)
```

Comment:
Maybe a note on users that might have custom packs / plugins that they will lose them?

Reply:
added thanks